theme/static/style/style.css

```diff
@@ -400,11 +426,15 @@ pre code {
 
 
     /* Screenshots section */
-    .screenshots .img-index {
+    .screenshots .col-2, .img-index {
         display: none;
         width: 0;
     }
 
+    .screenshots .col-8 {
+        width: 100%;
+    }
+
```

Comment:
Move this back to the top from where you have deleted it. It controls the styling of the screenshots image in the main page and for screens under 1360px we still want to have the image span the full width. With the current change the images goes on to the left which is a regression from existing behavior.